### ISSUE TITLE
[WinUI 3 Gallery: Scrolling -> SemanticZoom]: Keyboard focus is going to the 'Static' text under 'A Simple SemanticZoom' section.

### ISSUE THREAD

**Issue by sabareesanms** [microsoft/WinUI-Gallery#1944]
**Environment Details:**

**OS Version:** 24H2 (OS Build 26100.4315)

**App:** WinUI 3 Gallery

**App Version:** 2.6.0.0 (SDK 1.7)

**AT:** Keyboard

**Repro Steps:**

1. Launch the WinUI 3 Gallery app.

2. Navigate to search box using tab key.

3. Type 'SemanticZoom' in search box and hit enter.

4. Now navigate inside the static text under  'A Simple SemanticZoom' section using arrow keys.

5. Now observe the behavior

**Actual Result:**

Keyboard focus is going to the 'Static' under 'A Simple SemanticZoom' section and on invoking it no action is getting performed.

**Expected Result:**

Keyboard focus should not go to the 'Static' text under 'A Simple SemanticZoom' section. Only interactive elements should be focusable.

**User Impact:**

Users with motor impairment who rely on keyboard will face difficulty if focus is going to the static text.

**Comment by niels9001** [comment]
This is by design, as this is a SemanticZoom control that will trigger an action (zooming out) when clicking the text:

![Image](https://github.com/user-attachments/assets/25f9a236-331c-4718-a960-464dcc1d28a6)
